client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -56,7 +56,14 @@ const MarketplaceThankYou = ( { productSlug }: { productSlug: string } ) => {
 	const siteId = useSelector( getSelectedSiteId );
 	const siteSlug = useSelector( getSelectedSiteSlug );
 	const isRequestingPlugins = useSelector( ( state ) => isRequesting( state, siteId ) );
-	const pluginOnSite = useSelector( ( state ) => getPluginOnSite( state, siteId, productSlug ) );
+
+	// retrieve WPCom plugin data
+	const { data: wpComPluginData } = useWPCOMPlugin( productSlug );
+	const softwareSlug = wpComPluginData
+		? wpComPluginData.software_slug || wpComPluginData.org_slug || wpComPluginData.slug
```

Comment:
`|| wpComPluginData.slug` is this a fallback? Cause we have already migrated all `slug` to `software_slug` in `Marketplace_Product_Catalog` and I think we can safely remove it.

Reply:
Yeah, it was meant to be a fallback, but good to know. I'll remove it, thanks!